sdk/core/azure-core-amqp/src/test/java/com/azure/core/amqp/implementation/ReactorSenderTest.java

```diff
@@ -117,6 +122,30 @@ public void testLinkSize() throws IOException {
         verify(sender, times(1)).getRemoteMaxMessageSize();
```

Comment:
There are no unit tests here that actually test the complex logic in your ReactorSender operation.. (which will eventually moved to TransactionCoordinator)